src/map/grid_map.rs

```diff
@@ -53,6 +53,21 @@ where
     /// If the point lies exactly on a cell boundary, the higher cell is returned.
     #[inline]
     fn cell_from_point(&self, point: &Point<NaD>) -> Cell<NaD>;
+
+    #[inline]
+    fn track_changes(&self) -> bool;
+
+    #[inline]
+    fn set_track_changes(&mut self, value: bool);
+
+    #[inline]
+    fn changed_cells(&self) -> Vec<Cell<NaD>>;
+
+    #[inline]
+    fn clear_changed_cells(&mut self);
+
+    #[inline]
+    fn add_changed_cells(&mut self, cells: Vec<Cell<NaD>>);
```

Comment:
I think this should be done in the GridMap::set method. It could just check if the new value is equal to the old value and add it to changed_cells if they're different. You'll probably need to make this a Set instead of a Vector too, so that it doesn't have duplicates for the same cell, if it's updated more than once in between calls to clear_changed_cells.

Reply:
didn't change as this will be coming through the network at some point